File: articles/metrics-objectives-and-goodharts-law/typescript/goodhart_risk.ts
const proxyGap = 0.38;
const pressure = 0.88;
const gaming = 0.72;
const guardrailPenalty = 1.0;
const score = (proxyGap + pressure + gaming + guardrailPenalty) / 4.0;
console.log(`goodhart_risk_score=${score.toFixed(4)}`);

File: articles/metrics-objectives-and-goodharts-law/haskell/GoodhartRisk.hs
module Main where

main :: IO ()
main = do
  let proxyGap = 0.38
      pressure = 0.88
      gaming = 0.72
      guardrailPenalty = 1.0
      score = (proxyGap + pressure + gaming + guardrailPenalty) / 4
  putStrLn ("goodhart_risk_score=" ++ show score)
